media_softlet/linux/common/os/i915/include/mos_bufmgr_priv.h

```diff
@@ -358,9 +358,9 @@ struct mos_bufmgr {
     void (*disable_object_capture)(struct mos_bufmgr *bufmgr) = nullptr;
     int (*get_memory_info)(struct mos_bufmgr *bufmgr, char *info, uint32_t length) = nullptr;
     int (*get_devid)(struct mos_bufmgr *bufmgr) = nullptr;
+    void (*realloc_cache)(struct mos_bufmgr *bufmgr, uint8_t alloc_mode) = nullptr;
```

Comment:
Should have same function pointer definition for other kernel version?

We have multiple mos_bufmgr_priv.h headers existing in source

Reply:
are you referring to i915_production?

Reply:
Yeah, they share same mos_bufmgr_api.c, it may have compiling issue when you enable production kmd.

Reply:
Aligned the API production kernel too.